packages/op-datasets/src/op_datasets/etl/intermediate/models/daily_address_summary.py

```diff
@@ -171,63 +192,51 @@
         alias="avg_l1_gas_price_gwei",
         sql_expr=wei_to_gwei(
             safe_div(
-                "SUM(receipt_l1_gas_price * receipt_l1_gas_used)",
-                "SUM(receipt_l1_gas_used)",
+                f"SUM({L1_CONTRIB_L1_GAS})",
+                f"SUM(({ESTIMATED_SIZE}) * COALESCE(16*receipt_l1_base_fee_scalar/1000000, receipt_l1_fee_scalar))",
```

Comment:
i wonder if this can be replaced by a standard variable function?

Like, this is `L1_Contrib_L1_Gas/receipt_l1_gas_price` - which also makes it easier to communicate that this is a weighted average.

Similar idea for `L1_CONTRIB_BLOB/receipt_l1_blob_base_fee`

Reply:
Not sure if i fully get this, but i suppose abstracting away the denominator into something would help readability here